components/formats-bsd/src/loci/formats/in/JPEGReader.java

```diff
@@ -207,8 +211,28 @@ protected void initFile(String id) throws FormatException, IOException {
         }
 
         HashMap<String, String> tags = exif.getTags();
-        for (String tagName : tags.keySet()) {
-          addGlobalMeta(tagName, tags.get(tagName));
+        for (String tagName : tags.keySet()) {  
+          if (tagName.equals(exif.getTagName(ExifSubIFDDirectory.TAG_X_RESOLUTION))) {
```

Comment:
I had removed Resolution Unit from the previous iteration of the PR as the documented options for it were inches or cm, which didn't really seemed tailored to this application. We could add it back in again though.